src/main/groovy/edu/oregonstate/mist/slashcourses/core/SlashCourse.groovy

```diff
@@ -0,0 +1,22 @@
+package edu.oregonstate.mist.slashcourses.core
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+
+
+/**
+ * Slash Course Class
+ */
+
+class SlashCourse {
+    Integer crn
```

Comment:
The fields / properties in this class should use: camelCase instead of under_scores.